apps/src/code-studio/pd/application/teacher/ChooseYourProgram.jsx

```diff
@@ -72,16 +83,17 @@ const ChooseYourProgram = props => {
       (csHowManyMinutes * csHowManyDaysPerWeek * csHowManyWeeksPerYear) / 60;
   }
 
-  let belowMinCourseHours = false;
-  let minCourseHours = programInfo.minCourseHours;
-  if (courseHours !== null && courseHours < minCourseHours) {
-    belowMinCourseHours = true;
-  }
-
```

Comment:
This logic is being removed here since the text down on line 195 in the original is being removed and that is the only spot `belowMinCourseHours` is referenced in this file